noxfile.py

```diff
@@ -0,0 +1,40 @@
+import nox
+
+package = "dbt-hive"
+python_versions = ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
How do we ensure that all these python versions are installed in the machine?

Reply:
We can't. Users need to install these versions of python on their machine. Tox throws a warning if the version is not present.